packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -1,119 +1,113 @@
-import React from 'react'
-import {DatePicker, FormGroup, ControlLabel, Form} from 'rsuite'
+import React, {useState} from 'react'
+
+import './dateTimePicker.less'
+
+import DatePicker from 'react-datepicker'
+import {ControlLabel, Button, ButtonGroup, ButtonToolbar} from 'rsuite'
+
 import {useTranslation} from 'react-i18next'
 
-export interface DateTimeRange {
+export interface DateTimePreset {
   label: string
-  value: Date
+  offset: number
 }
 
-export interface DateTimePickerProps {
+export interface KeyboardDateTimePickerProps {
   dateTime: Date | undefined
   label: string
+  changeDate(publishDate: any): void
```

Comment:
why we use `any` here? can we change it to the proper type?

Reply:
As best I understand it can't just be a Date type because that function allows any keyboard input. I haven't found how to define the type here in a more specific way that won't also cause type errors in other places.

Reply:
but if it can be a date or keyboard input then you could set the type to `Date | string`.